### DISCUSSION TITLE
Alpaca is ready for GNOME 47 🎉

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#175] [Announcements]
Hi everyone! I started using Gnome Nightly as my "test vm" for Alpaca, I've been tweeking the appearance of the app to use more of the systems accent color, with Gnome 47 users can now change the blue accent for a different color and I'm happy to report that it works with Alpaca along with all the new Adwaita changes!

I love how it looks, special thanks to the GNOME / Adwaita guys <3

![Screenshot From 2024-07-31 21-10-58](https://github.com/user-attachments/assets/1d5a836f-674a-4dac-9448-81f6e4d3f293)
![Screenshot From 2024-07-31 21-12-12](https://github.com/user-attachments/assets/f927e1ac-dc1a-4e5a-a3a3-8a9c769a9640)
![Screenshot From 2024-07-31 21-14-59](https://github.com/user-attachments/assets/57bc9140-9b3e-4f03-a481-3059d6e3fe75)

**Comment by Jeffser** [comment]
don't worry about the IP, it's not my public IP hahaha

**Comment by olumolu** [comment]
Download model button is not  that easy to understand i was think this should be replaced with a self explanatory icon so users can click there and download.
The icon that you have added for download model is not that good.

**Reply by Jeffser** [reply]
which one? the brain thingy?

**Reply by olumolu** [reply]
Can be also a basic download logo can do this.

**Comment by Jeffser** [comment]
So it turns out I shouldn't change the color for the chat titles to the accent color according to  GNOME HIG

My bad haha
